solr/core/src/java/org/apache/solr/search/SolrQueryTimeoutImpl.java

```diff
@@ -42,23 +44,23 @@ public static SolrQueryTimeoutImpl getInstance() {
   }
 
   /**
-   * Get the current value of timeoutAt.
+   * The time (nanoseconds) at which the request should be considered timed out.
    */
-  public static Long get() {
+  public static Long getTimeoutAtNs() {
```

Comment:
For back-compat sake, on 8x I could retain the former method as Deprecated... but I'm unsure if anyone calls it.